OpenRA.Game/Graphics/IGraphicsDevice.cs

```diff
@@ -55,6 +55,7 @@ public interface IGraphicsDevice : IDisposable
 		IShader CreateShader(string name);
 
 		Size WindowSize { get; }
+		void ChangeRes(Size size, WindowMode windowMode);
```

Comment:
`SetWindowSize(Size size)` please.  This doesn't use the `windowMode` arg, and there's no need to shorten "resolution" to "res" everywhere